File: Curso Java Com POO/Java com POO/Aula11/src/teste/Tecnico.java
package teste;

public class Tecnico extends Aluno {
    //Atributos
    private String registroProfissional;
    
    //Métodos
    public void praticar() {
        System.out.println("Técnico " + this.nome + " está praticando!");
    }
    
    //Getters e Setters
    //registroProfissional
    public String getRegistroProfissional() {
        return registroProfissional;
    }
    public void setRegistroProfissional(String registroProfissional) {
        this.registroProfissional = registroProfissional;
    }
    
    //Método toString
    @Override
    public String toString() {
        return "Aluno Tecnico{" + "nome=" + nome + ", idade=" + idade + ", sexo=" + sexo + 
                " matricula=" + matricula + ", curso=" + curso +
                " registroProfissional=" + registroProfissional + '}';
    }
    
}

File: Curso Programação Orientada a Objetos/PHP com POO/Aula11/Tecnico.php
<?php

require_once 'Aluno.php';

class Tecnico extends Aluno{
    //Atributos
    private $registroProfissional;
    
    //Métodos
    public function praticar() {
        echo "Técnico " . $this->registroProfissional . " está praticando!";
    }
    
    //Getters
    public function getRegistroProfissional() {
        return $this->registroProfissional;
    }

    //Setters
    public function setRegistroProfissional($registroProfissional) {
        $this->registroProfissional = $registroProfissional;
    }


}
